packages/webpack-config/webpack/loaders/createBabelLoaderAsync.js

```diff
@@ -75,18 +67,18 @@ module.exports = async function({
 
     include(inputPath) {
       for (const possibleModule of modules) {
-        if (inputPath.includes(possibleModule)) {
+        if (inputPath.match(possibleModule)) {
           if (verbose) {
             const packageName = packageNameFromPath(inputPath);
             logPackage(packageName);
           }
-          return inputPath;
+          return true;
         }
       }
       // Is inside the project and is not one of designated modules
-      if (inputPath.includes(ensuredProjectRoot)) {
+      if (inputPath.match(ensuredProjectRoot)) {
         for (const excluded of excludedRootPaths) {
-          if (inputPath.includes(excluded)) {
+          if (inputPath.match(excluded)) {
```

Comment:
this doesn't look like it should have changed

Reply:
matching against regex now